docs/source/AdministratorGuide/ExternalsSupport/index.rst

```diff
@@ -7,33 +7,17 @@ Supported environments and packages
 OS:
 ---
 
-DIRAC *server* installation is officially supported only for:
-
-- CERN CentOS 7 (`CC7 <https://linux.web.cern.ch/centos7/>`_)
-
-Tests are also regularly run for other platforms but no official support is offered for them.
-
-
-Python versions:
-----------------
-
-DIRAC brings its own version of python at installation time: this version is right now fixed to 2.7.13.
-An experimental support for python3 server installations is also offered from DIRAC version 7.3.
-
-DIRAC installation scripts (including pilots) use the native python version, and for this the python versions supported are:
-
-- 2.6.9 (native on SLC6)
-- 2.7.5 (native on CC7)
-- 2.7.13
-- 3.6.8+
+DIRAC *server* installation is possible for all architectures supported by DIRACOS2 releases: https://github.com/DIRACGrid/DIRACOS2/releases.
```

Comment:
This isn't the case, Andrei added macOS DIRACOS2 installers which are missing some dependencies that are needed for servers (most notably `tornado`). Non x86_64 installers are also missing htcondor. See the lines with ` # [` comments which filter out packages from some platforms: https://github.com/DIRACGrid/DIRACOS2/blob/main/construct.yaml#L30-L103

(This is relatively easy to fix but it's not a priority)

```suggestion
DIRAC *server* installation supported for most x86_64 Linux installations using `DIRACOS2 <https://github.com/DIRACGrid/DIRACOS2/releases>`_. Other architectures and platforms may work as a server installation however this is on a best effort basis and is not regularly tested.
```

I say most as it will fail on non-glibc based systems like Alpine Linux.